spec/factories/plos_papers.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :plos_paper do
+    title "A PLOS Paper"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.